### PR TITLE
Meth nerf

### PR DESCRIPTION
**Pull request by MenacingManatee**
### Intent of your Pull Request

Tweaks to meth to make it less stonks
#### Changelog

:cl:  
tweak: Meth speed lowered from 2 to 1.6 (previously version of PR had 1.3)
tweak: Meth now requires 1 part facid to increase complexity a bit
rscadd: Meth now creates a small caustic smoke cloud on reaction
/:cl:

### CODE REVIEW

**Comment by Moja10** [comment]
The rate that mannitol fixes brain damage is a fixed 2, and the original rate of meth brain damage was between 1 - 4, meaning that each brain tick you lost more brain health than you gained from mannitol, meaning that meth exclusively has to be used in short bursts before it just fucks your brain bad. You can't even stomach the brain damage without mannitol because you'll just end up in the tier of brain damage where you get permanent disabilities and brain traumas. This occurs about 40 seconds to a minute on meth, What this PR is basically doing is making the brain damage so random, extreme, and unpredictable that it's basically nigh on unusable, because if the RNG is bad you will basically get brain traumas that requires surgery to fix, if you aren't also lugging around a supply of neurine to deal with the neurine fixable brain traumas. Also keep in mind that you literally die at 200 brain damage which is possible through casual meth use in this PR, if RNG is bad. 

Meth is also useless for combat because you drop things on meth.

**Comment by Readystorm546** [comment]
> Meth is also useless for combat because you drop things on meth.

BAHAHHAHAHAHAHHAHAHHAH
Your wrong

**Comment by ynot01** [comment]
meth fun
pr anti-fun

**Comment by BOBsHERE** [comment]
I think anything thats strong and that a good player can effectively utilize should be nerfed or removed -yogs 2020.

**Comment by yacabo** [comment]
I really don't think some chemist should be capable of being as powerful as a ninja

**Comment by Moja10** [comment]
Correction, ephedrine makes you drop things and meth doesn't, despite meth using ephredine as a component. Maybe that could be added instead of these changes.

**Comment by BOBsHERE** [comment]
> I really don't think some chemist should be capable of being as powerful as a ninja

lol no ninja is dying to a meth chemist.

**Comment by Readystorm546** [comment]
> meth fun
> pr anti-fun

Being faster then  a jetpack "fun"

**Comment by BOBsHERE** [comment]
Making you drop things isnt nerfing it, its removing it entirely. Literally wont get used. Meth stun resistance is already bad compared to before in that it does not whatsoever prevent you from getting cuffed off of a stun.

**Comment by MenacingManatee** [comment]
Moja makes a decent point about the damage. With a max of 6, with improbably godawful rng you could go above 100 brain damage with less than you need to get addicted while taking mannitol. Now you have to take more to hit that point

**Comment by MenacingManatee** [comment]
I don't like the idea of it making you drop it just because I do agree that would likely fall under nerfing it out of existence. I'm not trying to be anti-fun with this PR, but meth is really unbalanced and isn't that hard to make

**Comment by BOBsHERE** [comment]
how about making meth ridiculously more complex to make, like skewium, to reflect its strength instead of just nerfing the chemical?

**Comment by MenacingManatee** [comment]
I still think meth needs to be slower than it is now, but what would you suggest as a component? If it was at least more difficult to make than 3 basic chems and ephedrine it could be justifiable to be so strong

**Comment by BOBsHERE** [comment]
i think that's the healthiest direction to balance it right now. Even for me making skewium is not fast. Meth should be at least as complex.

**Comment by BOBsHERE** [comment]
You could do literally anything to make it more complex to make. You could make a new shit chemical that has the same recipe as meth now, make it do something slightly better then ephedrine but still not good, then start adding steps from then. Like adding Diphenhydramineand atropine. Thats a crazy amount of base chemicals and steps right there while still keeping the risk of explosion.

**Comment by Hopekz** [comment]
This PR basically combats this stupidness.
This is real-time with podperson slowdown


https://cdn.discordapp.com/attachments/134722036353204224/703837575999389716/NZuIUpnhh0.gif

**Comment by MenacingManatee** [comment]
So here's my thought. irl meth releases caustic gas. I could do less of a nerf to speed, and remove the extra damage, if meth also took facid and released caustic smoke when created. Should increase the risk to make I want, and add complexity, and in return it could be stronger

**Comment by BOBsHERE** [comment]
> So here's my thought. irl meth releases caustic gas. I could do less of a nerf to speed, and remove the extra damage, if meth also took facid and released caustic smoke when created. Should increase the risk to make I want, and add complexity, and in return it could be stronger

so increase complexity, and make it more obvious you're making meth like when you make clf3.

**Comment by MenacingManatee** [comment]
Exactly, with a similar environmental risk like clf3, except acid instead of fire, with the added benefit of being slightly more similar to irl meth

**Comment by BOBsHERE** [comment]
definitely dont make it produce acid smoke, because that will melt all your clothes and everything in the room. but a small cloud of regular smoke, colored weird would be fine. Something that says hey the chemists are making super meth.

**Comment by MenacingManatee** [comment]
EDIT:
New changes:
Meth brain damage set to original value
Meth speed lowered from 2 to 1.6 (previously 1.3)
Meth now requires 1 part facid to increase complexity a bit
Meth now creates a small caustic smoke cloud on reaction

**Comment by MenacingManatee** [comment]
@Moja10 @BOBsHERE  added the changes we talked about

**Comment by MenacingManatee** [comment]
There is a bit of acid in the smoke, but its only 5u per reaction in a very small circle. Unless you just stand in it, it shouldn't damage you, and unless you make quite a bit it shouldn't hurt anything around you

**Comment by alexkar598** [comment]
complexity != balance, that's stupid, xenobio is arguably complex but that didnt make it anywhere near balanced.

**Comment by yacabo** [comment]
Show me the 1.6 speed, I'm not convinced that's enough

**Comment by BOBsHERE** [comment]
> complexity != balance, that's stupid, xenobio is arguably complex but that didnt make it anywhere near balanced.

Complexity absolutely has a major factor in balance. Time to acquire. Complexity and skill required. Amount able to be made. If you put meth behind a uranium paywall that would also severely limit its strength. Theres so many ways to balance things.

**Comment by MenacingManatee** [comment]
To be fair, 1.6 isn't quite enough imo. I'd prefer it ~1.2-1.4, but I'm trying to reach a compromise with the people who like it as it is now and the higher rp standards we're moving towards. 1.6 is a step in the right direction, and the increased complexity, danger, and obviousness helps to justify that compromise on the speed

**Comment by ghost** [comment]
what kind of retard makes meth and not bath salts

**Comment by MenacingManatee** [comment]
Bath salts don't give any movement speed

**Comment by MenacingManatee** [comment]
Not sure if it'll get merged with how controversial it is, but PR is finalized and ready for review

**Review by Hopekz — APPROVED** [review]
Code checks out

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 1]
```diff
@@ -24,10 +24,22 @@
...
+	var/datum/reagents/R = new/datum/reagents(5)
+	R.my_atom = src
+	R.add_reagent(/datum/reagent/toxin/acid, 5)
+	var/smoke_radius = round(0.5, 0.5)
+	var/location = get_turf(holder.my_atom)
+	var/datum/effect_system/smoke_spread/chem/S = new
+	playsound(location, 'sound/effects/smoke.ogg', 50, 1, -3)
+	if(S)
+		S.set_up(R, smoke_radius, location, 0)
+		S.start()
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/drugs.dm]:
```suggestion
/datum/chemical_reaction/methamphetamine/on_reaction(datum/reagents/holder)
	var/datum/reagents/R = new/datum/reagents(5)
	R.my_atom = holder.my_atom
	R.add_reagent(/datum/reagent/toxin/acid/fluacid, 5)
	var/datum/effect_system/smoke_spread/chem/S = new
	S.set_up(R, 0.5, holder.my_atom, 0)
	S.start()
	
	playsound(location, 'sound/effects/smoke.ogg', 50, 1, -3)
```
Couple of fixes to make the code look prettier and also somewhat more operational

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/recipes/drugs.dm (hunk 1 above)
Comment (MenacingManatee) [reply code/modules/reagents/chemistry/recipes/drugs.dm]: Added the requested changes

**Comment by BOBsHERE** [comment]
The smoke cloud lasts 10-15 seconds. It stays wayyyyy too long.

**Comment by BOBsHERE** [comment]
The speed nerf is actually a buff lol, it makes it much easier to control. 
It would be nice if you could make the tickrate slightly slower, or if it actually got some stun resistance back .

**Review by alexkar598 — APPROVED** [review]

**Comment by Marmio64** [comment]
i think it'd be cool to make meth inflict a shit ton of stamina damage on you depending on how much you moved while it's in you, and the affliction happens after the meth has metabolized. Because I don't know about anyone else, but running twice as fast as usain bolt for a minute would tire you the fuck out once the drugs left your system. Also I don't know about the whole smoke thing.

However, I don't play chemistry or use it like at all so *shrug*

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 2]
```diff
@@ -194,9 +194,10 @@
 	addiction_threshold = 10
 	metabolization_rate = 0.75 * REAGENTS_METABOLISM
 
+
```
Comment (JamieD1) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: Random space

Comment (MenacingManatee) [reply code/modules/reagents/chemistry/reagents/drug_reagents.dm]: Fixed

**Pull request merged**